### ISSUE TITLE
Hyperparameters and model for libero-plus training

### ISSUE THREAD

**Issue by lidc54** [MergeVLA/MergeVLA#2]
Could you please provide the hyperparameter settings used for training on libero-plus? Are they the same as those used for libero, or are there different configurations? It would be fine to share a temporary or preliminary version for now.  

Also, why is the test accuracy completely zero after training on libero-plus? Furthermore, when checking the rollout videos, there are no meaningful actions being performed at all.

**Comment by Daisy-zzz** [comment]
There is **no training on libero-plus**, we directly evaluate model trained on libero data in libero-plus environment for ood performence.

**Comment by missTL** [comment]
Does the training data include libero_90?

**Comment by YuxiaFu** [comment]
For the training of Libero, we only used the libero_spatial, libero_object, libero_goal, and libero_10 datasets.
